Makefile

```diff
@@ -434,7 +428,11 @@ catalog-build-replaces: opm ## Build a catalog image using replace mode
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 
-OADP_BUCKET = $(shell cat $(OADP_BUCKET_FILE))
+.PHONY: install-ginkgo
+install-ginkgo: # Make sure ginkgo is in $GOPATH/bin
```

Comment:
@kaovilai since we are discussing it

Do you know how to install this in `ProjectRoot/bin/` instead of `$GOPATH/bin`?
(we can address this later in another PR as well)

Reply:
Yes. The code for this is already in the makefile under `go-install-tool`.

Here's an example: https://github.com/kaovilai/oadp-operator/blob/b09ce0461bc113e3dbb409d52e38b46555e808dc/Makefile#L177-L178

Reply:
Do you wanna do go-install-tool for ginkgo here? or in a later PR?

Reply:
will try on this later. If its simple, will do in this one; otherwise, another

Reply:
It's very simple like you couldn't believe

Reply:
Alright, tested and works, but I didn't like we have to hardcode version (if we update ginkgo version, we need to update go.mod and makefile)

What you think? I am ok hardcoding it, as I believe ginkgo errors out if executing with different version

Reply:
> I didn't like we have to hardcode version

Like we have to pick a version rather than using latest?

You could @latest as well but we wanna minimize external changes to CI. We should bump versions via explicit PRs IMO.

You could get CLI based on go mod version. like here
https://github.com/mateusoliveira43/oadp-operator/blob/fix/testing-instructions/build/ci-Dockerfile#L8C5-L8C5

```Dockerfile
RUN go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
```

This would install the same CLI as specified in go.mod

Reply:
Might need to update go-install-tool to accept -mod=mod

Reply:
we could use latest (already outdated, we are on `2.11.0` and latest is `2.12.1`) and update when needed (do not like this approach because the update necessity will be when CI start breaking and not when we want)

Yeah, was using this approach, but then it install computer wide ($GOPATH/bin instead of PROJECT_ROOT/bin)

Reply:
going to try to modify go-install-tool , if I can not, going to add a TODO

Reply:
alright, got it working
```make
# go-install-tool-mod will 'go install' any package $2 and install it to $1 with same version as from Project's go.mod.
PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
define go-install-tool-mod
@[ -f $(1) ] || { \
set -e ;\
TMP_DIR=$$(mktemp -d) ;\
cd $$TMP_DIR ;\
cp $(PROJECT_DIR)/go.mod . ;\
echo "Downloading $(2)" ;\
GOBIN=$(PROJECT_DIR)/bin go install -mod=mod $(2) ;\
rm -rf $$TMP_DIR ;\
}
endef
```

But then, this https://github.com/openshift/oadp-operator/blob/master/build/ci-Dockerfile#L8 would be unnecessary, right? 

If so, I propose we do this in a separate PR to also change Dockerfile